src/instructlab/configuration.py

```diff
@@ -586,13 +793,209 @@ def ensure_storage_directories_exist():
         "lora_target_modules": ["q_proj", "k_proj", "v_proj", "o_proj"],
     }
 
+    internal_config = Config(
+        chat=_chat(model="~/.cache/instructlab/models/granite-7b-redhat-lab"),
+        serve=_serve(model_path="~/.cache/instructlab/models/granite-7b-redhat-lab"),
+        generate=_generate(
+            model="~/.cache/instructlab/models/mixtral-8x7b-v0-1",
+            gpus=4,
+            teacher=_serve(
+                backend="vllm",
+                model_path="~/.cache/instructlab/models/mixtral-8x7b-v0-1",
```

Comment:
Is it alright that we are using `~` here? Will that work on non-Linux platforms?

Reply:
It won't. I pointed it out elsewhere.

Reply:
it works on mac / linux- those are all we're targeting right?

Reply:
AFAIU macosx does not use XDG paths. It puts cached models under Application Support directory. (I think there's an issue somewhere to understand if this should be changed to ~/.cache though.)

Reply:
(I also don't know if we explicitly target **only** *nix platforms. In theory, this is just python code that probably runs on win32 etc.? **I don't know if we care though**.)

Reply:
Here's the macosx bug: https://github.com/instructlab/instructlab/issues/1747

Reply:
fixed